### PR TITLE
Fix DraftJS link parsing issue

### PR DESCRIPTION
**Pull request by vigneshTheDev**

### CODE REVIEW

**Comment by vikasrohit** [comment]
@maxceem lets merge this in dev only as regular fix. I think we can afford the current issue as it is not limiting users to use our platform in any way. This is additional functionality that works partially (i.e. when you paste the link only it works, right?). So, I guess we should be good to have it as regular fix.

**Comment by maxceem** [comment]
@vikasrohit yes, makes sense.

File: src/components/RichTextArea/LinkPlugin/Link/Link.jsx [hunk 1]
```diff
@@ -24,9 +24,22 @@ export default class Link extends Component {
...
+    this.removeElementGetter(contentState, entityKey)
+  }
+
   setElementGetter(contentState, entityKey) {
     contentState.mergeEntityData(entityKey, {
-      el: () => this.element
+      el: () => {
+        return this.element
+      }
+    })
```
[the remark this answers is not in the document]
Comment (vigneshTheDev) [reply src/components/RichTextArea/LinkPlugin/Link/Link.jsx]: Yes. Correct. I changed to enable debugging. I'll make the change and push it

[the remark this answers is not in the document]
Comment (vigneshTheDev) [reply src/components/RichTextArea/LinkPlugin/Link/Link.jsx]: Hi @maxceem, looks like the pull request is closed. Shall I submit a new PR with the change?

**Pull request merged**